tests/kafkatest/services/kafka.py

```diff
@@ -96,7 +100,7 @@ def stop_node(self, node, clean_shutdown=True):
 
     def clean_node(self, node):
         node.account.kill_process("kafka", clean_shutdown=False, allow_fail=True)
-        node.account.ssh("rm -rf /mnt/kafka-logs /mnt/kafka.properties /mnt/kafka.log /mnt/kafka.pid", allow_fail=False)
+        node.account.ssh("rm -rf /mnt/kafka-logs /mnt/kafka.properties /mnt/kafka.log /mnt/kafka.pid /mnt/ssl", allow_fail=False)
```

Comment:
See other comments about how `SecurityConfig` is used and whether a mixin would make sense. Seems less than ideal that we have to scatter `/mnt/ssl` throughout all the `clean-node` methods.